test/issue1531-min-dub-version.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+set -e
+
+. $(dirname "${BASH_SOURCE[0]}")/common.sh
+
+DIR1531=${CURR_DIR}/issue1531-min-dub-version
+
+rm -rf $DIR1531/.dub/
+rm -rf $DIR1531/test-application*
+
+sed -i 's/^minDubVersion ".*"$/minDubVersion "1.0"/' ${DIR1531}/dub.sdl
```

Comment:
It's not a good idea to mutate files that are checked in the source tree by the testsuite, better put the `dub.sdl` into a temporary directory.
Also since the last version dub supports passing files in from stdin, so you could use `--single` plus pass the file in via stdin which would eliminate the need for a temporary directory entirely.